change/@fluentui-react-9389c81c-3bf8-4fd7-a465-31fd4a677c80.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix(Breadcrumb): overflowbutton focus outline now visible in HCM.",
```

nit: based on the RFC I think we only need the `<tag>: <message>` without the `(<affected package>)`. Feel free to ignore though

```suggestion
  "comment": "fix: overflowButton focus outline now visible in HCM.",
```